utils/browser-version-has-release-date.mjs

```diff
@@ -1,13 +1,14 @@
 import caniuse from 'caniuse-lite';
+import semver from 'semver';
```

Comment:
`semver` is really handy when dealing with version strings.
Removes a bunch of regexp and other custom logic if we embrace this.

Dev dependency only.